tiri/src/lib.rs

```diff
@@ -45,12 +42,13 @@ use yktrace::tir::{
 #[allow(dead_code)]
 const DUMP_CONTEXT: usize = 4;
 
-/// Each local variable is represented by one `Value` struct.
+/// The contents of each local variable is represented by a `Value`.
 #[derive(Debug, Clone, Eq, PartialEq)]
 enum Value {
-    /// A stand-alone immediate value.
+    /// A value that can be stored immediately into a local variable without touching memory.
```

Comment:
I think this might need to be something like "A `Value` which is small enough that it can potentially be stored in 1 or more registers."? I'm not sure if the "potentially" thing is true or not though!

Reply:
I'm trying to understand what the difference between these two things is, so I'm trying to find a description you're unhappy enough with that you'll fix to something better :)

There is no machine I know of that has 4GiB of registers, so that's why I wondered if the "fits in one or more registers" things might be the right route. I'm not saying it is, but hopefully you can see what I'm getting at!